compiler/plugins/target/AMD-AIE/iree-amd-aie/Target/AIETarget.cpp

```diff
@@ -673,12 +674,20 @@ LogicalResult generateXCLBin(MLIRContext *context, ModuleOp moduleOp,
     }
     SmallVector<char, 0> kernelsJsonDataString;
     llvm::raw_svector_ostream ostream(kernelsJsonDataString);
-    std::string kernels_json_data = R"(
+    std::string kernels_json_data = "";
+    for (auto entryPointName : entryPointNames) {
+      // this print is useful for non-IREE test benches as they need the
+      // Kernel names
+      if (toolkit.isVerbose()) {
+        llvm::errs() << "Entry point name: " << entryPointName << "\n";
```

Comment:
Would it still work if this was wrapped in LLVM_DEBUG (like in https://github.com/nod-ai/iree-amd-aie/pull/40) or if it was prefixed with [AMD-AIE plugin]?